core/server/src/main/java/alluxio/proxy/StreamRestServiceHandler.java

```diff
@@ -33,13 +35,13 @@
 import javax.ws.rs.core.Response;
 
 /**
- * This class is a REST handler for file system client requests.
+ * This class is a REST handler for stream resources.
  */
 @NotThreadSafe
-@Path(StreamsRestServiceHandler.SERVICE_PREFIX)
+@Path(StreamRestServiceHandler.SERVICE_PREFIX)
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-public final class StreamsRestServiceHandler {
+public final class StreamRestServiceHandler {
```

Comment:
I liked streams better since the prefix is streams. Generally it seems nice to have the REST service handler for resource X be called XRestServiceHandler